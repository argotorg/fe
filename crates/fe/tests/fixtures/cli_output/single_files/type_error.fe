pub fn add_numbers() -> i32 {
    let x: i32 = 5
    let y: bool = true
    return x + y  // Type error: can't add i32 and bool
}