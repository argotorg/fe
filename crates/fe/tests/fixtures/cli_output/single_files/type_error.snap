---
source: crates/fe/tests/cli_output.rs
assertion_line: 77
expression: output
input_file: tests/fixtures/cli_output/single_files/type_error.fe
---
=== STDERR ===
errors in file://<project>/crates/fe/tests/fixtures/cli_output/single_files/type_error.fe

error[8-0000]: type mismatch
  ┌─ type_error.fe:4:16
  │
4 │     return x + y  // Type error: can't add i32 and bool
  │                ^ expected `i32`, but `bool` is given


=== EXIT CODE: 1 ===
